pkg/cloudprovider/providers/azure/azure_storageaccount.go

```diff
@@ -51,6 +51,9 @@ func (az *Cloud) getStorageAccounts() ([]accountWithLocation, error) {
 			storageType := ""
 			if acct.Sku != nil {
 				storageType = string((*acct.Sku).Name)
+				if matchingAccountType != "" && strings.ToLower(matchingAccountType) != strings.ToLower(storageType) {
```

Comment:
!strings.EqualFold

Reply:
Removed approved label.

@andyzhangx Could you fix this?

Reply:
fixed